scripts/binary/build.coffee

```diff
@@ -126,6 +125,20 @@ buildCypressApp = (platform, version, options = {}) ->
 
     packages.copyAllToDist(distDir())
 
+  copyPatches = ->
+    log("#copyPatches")
+
+    patchesPath = path.join(__dirname, '..', '..', 'patches')
+    destPatchesPath = path.join(distDir(), 'patches')
+
+    fs.mkdirSync(destPatchesPath)
+    fs.readdirSync(patchesPath)
+    .map((patchFileName) -> [
+      path.join(patchesPath, patchFileName),
+      path.join(destPatchesPath, patchFileName)
+    ])
+    .forEach(([src, dest]) -> fs.copyFileSync(src, dest))
+
```

Comment:
is there a reason for switching to use this `copyPatches` phase of the build rather than including the `patches` folder in `package.json#files` array ?